class.jetpack-cli.php

```diff
@@ -559,6 +559,184 @@ public function options( $args, $assoc_args ) {
 				break;
 		}
 	}
+
+	public function sync( $args, $assoc_args ) {
+		if ( ! Jetpack::is_active() ) {
+			WP_CLI::error( __( 'Jetpack must be connected to WordPress.com to sync.', 'jetpack' ) );
+		}
+
+		$action = isset( $args[0] ) ? $args[0] : 'status';
+
+		$allowed_actions = array(
+			'status',
+			'start',
+			'queue',
+		);
+
+		if ( ! in_array( $action, $allowed_actions ) ) {
+			WP_CLI::error( sprintf( __( '%s is not a valid command.', 'jetpack' ), $action ) );
+		}
+
+		switch ( $action ) {
+			case 'status':
+				require_once JETPACK__PLUGIN_DIR . 'sync/class.jetpack-sync-modules.php';
```

Comment:
Could we reuse some of the sync status endpoint code by calling from it directly. Since it almost looks identical?